File: Ejercicios Propuestos 2/Ejercicio 3/Ejercicio 3.jl
# Validación de entrada
while true
    println("Ingrese un número natural (mayor que 0):")
    n = parse(Int, readline())
    
    if n <= 0
        println("Error: debe ingresar un número natural positivo.")
    else
        global n
        break # Hasta Que n > 0
    end
end

suma = 0

for i in 1:1:n # "1:1:n" representa: desde 1 hasta n con paso 1
    global suma
    suma = suma + (i * i)
end

println("La suma de los cuadrados de los ", n, " primeros números naturales es: ", suma)

File: Ejercicios Propuestos 2/Ejercicio 3/ejercicio3.cpp
#include <iostream>

using namespace std;

int main() {
    // Definir n, i, suma Como Entero
    int n, i;

    long long suma; 
    
    // Validación de entrada
    do {
        cout << "Ingrese un número natural (mayor que 0):" << endl;
        cin >> n;
        
        if (n <= 0) {
            cout << "Error: debe ingresar un número natural positivo." << endl;
        }
    } while (n <= 0); // "Hasta Que n > 0" se traduce como "mientras n sea menor o igual a 0"
    
    suma = 0;
    
    // Para i <- 1 Hasta n Con Paso 1 Hacer
    for (i = 1; i <= n; i++) {
        suma = suma + (i * i);
    }
    
    cout << "La suma de los cuadrados de los " << n << " primeros números naturales es: " << suma << endl;
    
    return 0;
}